hack/cmds/commands_vectorsets.json

```diff
@@ -0,0 +1,254 @@
+{
+  "VADD": {
+    "summary": "Adds an element to a vector set, creating a new set if it doesn't already exist.",
+    "since": "Redis CE 8.0.0",
+    "group": "vector_set",
+    "complexity": "O(log(N)) for each element added, where N is the number of elements in the vector set.",
+    "arguments": [
+      {
+        "name": "key",
+        "type": "key"
+      },
+      {
+        "name": "dim",
+        "type": "integer",
+        "command": "REDUCE",
+        "optional": true
+      },
+      {
+        "type": "enum",
+        "command": "FP32",
+        "enum": ["FP32", "VALUES"],
+        "optional": true
+      },
+      {
+        "name": "num",
+        "type": "integer",
+        "optional": true
+      },
+      {
+        "name": "vector",
+        "type": "string"
+      },
```

Comment:
These don't seem correct. You should not allow "FP32" to be followed by "num".

Check this out:
<img width="685" alt="image" src="https://github.com/user-attachments/assets/5cf004aa-5cb2-43ec-a728-fe791a41fd70" />